### PR TITLE
Remove REST API and use a file instead

### PR DESCRIPTION
**Pull request by cmalangone**
Removed the REST API calls and changed to read the file produced by the platform-input-support.
Change the yaml config file

### CODE REVIEW

File: mrtarget/modules/ChEMBL.py [hunk 1]
```diff
@@ -3,41 +3,19 @@
...
-import more_itertools
+import shelve
+import dumbdbm
+import tempfile
 
-from mrtarget.common.ElasticsearchQuery import ESQuery
-from mrtarget.common.connection import new_es_client
-from mrtarget.Settings import Config
 from mrtarget.common import URLZSource
 import json
```
Comment (afaulconbridge) [inline mrtarget/modules/ChEMBL.py]: Better to do `import simplejson as json`. Its already a dependency elsewhere, and should give a performance boost.

File: mrtarget/modules/ChEMBL.py [hunk 2]
```diff
@@ -201,19 +194,13 @@ def _append_to_mol2syn(m2s_dict, molecule):
...
+        data = {'molecules':[]}
+        for mol_k in molecule_set:
+            if self.molecules_dict.has_key(mol_k):
+                data['molecules'].append(self.molecules_dict[mol_k])
+            else:
+                self._logger.error('problem retrieving the molecule info from the local db %s', str(mol_k))
+                raise ValueError('problem retrieving the molecule info from the local db', str(mol_k))
 
         #if the data is what we expected, process it
         if 'molecules' in data:
```
Comment (afaulconbridge) [inline mrtarget/modules/ChEMBL.py]: Given the key is being created just before, why is this code checking for it?

File: mrtarget/modules/ChEMBL.py (hunk 1 above)
Comment (cmalangone) [reply mrtarget/modules/ChEMBL.py]: It makes sense. Thanks

Comment (cmalangone) [reply mrtarget/modules/ChEMBL.py]: About shelve, Adam and I verified that computationally is better use shelve. Adding comment about why I added dumbdbm.

File: mrtarget/modules/ChEMBL.py (hunk 2 above)
Comment (cmalangone) [reply mrtarget/modules/ChEMBL.py]: The item must be present otherwise something is missing from ChEMBL molecule REST API.

**Pull request merged**